packages/shared/components/molecules/AssetIcon.svelte

```diff
@@ -72,7 +73,7 @@
                 classes={large ? 'w-12 h-12' : small ? 'w-6 h-6' : 'w-8 h-8'}
                 animation={ANIMATED_TOKEN_IDS[asset.id]}
```

Comment:
Right now it returns a `string`, but could you return an `AnimationEnum` instead, wdyt?